utils/libnetwork/Makefile

```diff
@@ -1,7 +1,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnetwork
-PKG_VERSION:=0.8.0-dev.2
+PKG_SOURCE_VERSION:=fc5a7d91d54cc98f64fc28f9e288b46a0bee756c
```

Comment:
It seems more often than not we are going to use commit id instead of tag to refer to a specific revision, I would suggest we stick to using `PKG_SOURCE_VERSION` for versioning.

Reply:
added `PKG_SOURCE_DATE`